challenge_7/python/tvanderwal/src/findMissingNumber.py

```diff
@@ -0,0 +1,12 @@
+#Challenge 7: Return a missing integer from an unsort list of integers
+
+def findMissingNumber(intList):
+    #sort list
+    intList.sort()
```

Comment:
Thanks! I actually just updated it to be O(n), but I think there is even a better algo to find the sum of the range over my for loop. Thanks for the feedback though. Sorry for the hard to test code, I was taking the README examples and just using them. Now I've been building them easier to use unittesting so hopefully that helps